src/components/_global/BalChart/BalChart.vue

```diff
@@ -61,7 +61,7 @@ type Props = {
   // eslint-disable-next-line vue/require-default-prop -- TODO: Define default prop
   chartClass?: string; // sets the class for the chart container
   // eslint-disable-next-line vue/require-default-prop -- TODO: Define default prop
-  wrapperClass?: string[]; // sets the class for the element which wraps the chart and the header
+  wrapperClass?: any; // sets the class for the element which wraps the chart and the header
```

Comment:
Because Vue allows `class` attribute to be any type, and here `wrapperClass` is just passed down to a `div` element. I used the same type as defined in: https://github.com/vuejs/vue/blob/main/types/jsx.d.ts#L239

Instead of setting it to `any` here, I could import that type directly from Vue, or alternatively define it as unknown `wrapperClass?: unknown`. Which one you'd think is better?